spotbugs/src/main/java/edu/umd/cs/findbugs/SAXBugCollectionHandler.java

```diff
@@ -300,7 +300,7 @@ public void startElement(String uri, String name, String qName, Attributes attri
                         try {
                             bugCollection.getProjectStats().setTimestamp(timestamp);
                         } catch (java.text.ParseException e) {
-                            throw new SAXException("Unparseable sequence number: '" + timestamp + "'", e);
+                            throw new SAXException("Unparsable sequence number: '" + timestamp + "'", e);
```

Comment:
That one is potentially debatable (or is it debateable? ;)

Reply:
Absolutely, let's keep it as is.